.github/workflows/ci_focal.yml

```diff
@@ -0,0 +1,47 @@
+name: CI - Ubuntu Focal
+
+on:
```

Comment:
Shouldn't the branch name `noetic-devel` be in here somewhere, or is this convention baked in into the setup script form ROS-I?

Reply:
Ok, I was not sure how this label was used. I'll merge it and see how it works and looks, and we can fine tune these details later :)